draft-song-dns-wireformat-http-01.txt

```diff
@@ -271,17 +274,17 @@ Internet-Draft          DNS wire-format over HTTP          February 2016
 
    Proxy-DNS-Transport: xyz
```

Comment:
Linjian Song (Davey) wrote:
> In draft-song-dns-wireformat-http-01.txt
> <https://github.com/songlinjian/DNS_HTTP/pull/6#discussion_r55784015>:
>
>>  @@ -271,17 +274,17 @@ Internet-Draft          DNS wire-format over HTTP          February 2016
>>
>>      Proxy-DNS-Transport: xyz
>
>     I think it should be mandatory.
>
> OK. BTW, Paul, would you like to be one of co-author with us ? or just a
> contributor.

if it's the protocol i developed, i'll be happy to be listed as a 
co-author, or not, as you please.

-- 
P Vixie